ui/raidboss/data/07-dt/raid/r1n.ts

```diff
@@ -1,14 +1,300 @@
+  },
+  '1C': {
+    'location': '1C',
+    ...mapEffectTileOverlay,
+  },
+  '1D': {
+    'location': '1D',
+    ...mapEffectTileOverlay,
+  },
+  '1E': {
+    'location': '1E',
+    ...mapEffectTileOverlay,
+  },
+  '1F': {
+    'location': '1F',
+    ...mapEffectTileOverlay,
+  },
+} as const;
+console.assert(mapEffectData);
+
+const headMarkerData = {
+  // Vfx Path: com_share1f
+  stack: '5D',
+  // Vfx Path: tank_lockon02k1
+  tankBuster: 'DA',
+  // Vfx Path: loc05sp_05a_se_p
+  lineStack: '178',
+} as const;
+console.assert(headMarkerData);
+
+// TODO:
+// Mouser
+// Predaceous Pounce
+// Leaping Black Cat Crossing
+
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM1',
   zoneId: ZoneId.AacLightHeavyweightM1,
   timelineFile: 'r1n.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'R1N One-two Paw Right Left',
+      type: 'StartsUsing',
+      netRegex: { id: '9309', source: 'Black Cat', capture: false },
+      durationSeconds: 9.5,
+      response: Responses.goRightThenLeft(),
+    },
+    {
+      id: 'R1N One-two Paw Left Right',
+      type: 'StartsUsing',
+      netRegex: { id: '930C', source: 'Black Cat', capture: false },
+      durationSeconds: 9.5,
+      response: Responses.goLeftThenRight(),
+    },
+    {
+      id: 'R1N Black Cat Crossing',
+      type: 'StartsUsing',
+      netRegex: { id: '930F', source: 'Black Cat', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Cards => Intercards',
+        },
+      },
+    },
+    {
+      id: 'R1N Elevate and Eviscerate',
+      type: 'StartsUsing',
+      netRegex: { id: '9317', source: 'Black Cat', capture: true },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Aim for pristine tile',
+        },
+      },
+    },
+    {
+      id: 'R1N Bloody Scratch',
+      type: 'StartsUsing',
+      netRegex: { id: '9340', source: 'Black Cat', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R1N Biscuit Maker',
+      type: 'StartsUsing',
+      netRegex: { id: '934A', source: 'Black Cat', capture: true },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'R1N Clawful',
+      type: 'StartsUsing',
+      netRegex: { id: '933C', source: 'Black Cat', capture: true },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'R1N Overshadow',
+      type: 'StartsUsing',
+      netRegex: { id: '9319', source: 'Black Cat', capture: true },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'R1N Leaping One-two Paw West West East',
+      type: 'StartsUsing',
+      netRegex: { id: '931F', source: 'Black Cat', capture: false },
+      durationSeconds: 10.5,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'West, West => East',
```

Comment:
These ones are a pain to write for. I almost wonder whether it's worth eating the length and doing something like "West => East at marker".

Reply:
They're almost the same (your suggestion is only 4 characters longer). I can't check right now, but if `West => East at marker` fits without wrapping I think that's fine?

Reply:
Hmm, yes. We definitely have used longer text strings in timelines before. My timelines on Gubal Hard and Dun Scaith both do, and so does quiqsuous's work on Aetherfront and Aloalo Island.  I know at least Dun Scaith and Aetherfront definitely don't wrap. I think we can just go with it for now and make adjustments as needed if it's an issue in the release?